src/Fist/Container/src/Container.php

```diff
@@ -348,7 +348,7 @@ protected function resolveNonClass(ReflectionParameter $parameter)
             return $parameter->getDefaultValue();
         }
 
-        $message = "Unresolvable dependency resolving [$parameter] in class {$parameter->getDeclaringClass()->getName()}";
+        $message = "Unresolvable dependency resolving [$parameter] in class {$parameter->getDeclaringClass()->name}";
```

Comment:
Switch to using `$parameter->getDeclaringClass()->name`. There is an [issue](https://bugs.php.net/bug.php?id=61384) with `getName()` and APC-enabled PHP versions.